### ISSUE TITLE
Category is not translated in Article View

### ISSUE THREAD

**Issue by tdzweb** [JoomFish/joomfish-2.5#88]
The article category is not translated in the item views. Tested with both Beez2 and Gantry templates. I have a fairly pristine installation of Joomla 2.5.6. Only installed the Gantry Framework bundle and the Quick Logout extension.

Test site (note, registration is enabled. I will give manager access if you ask):
http://dev1.tdzwebdesign.no/joomla25/j256-a/

Edit: Joomfish build: JoomFish-jf-future-alpha6-8-g743b4aa

**Comment by tdzweb** [comment]
A closely related issue - I suspect -  is that the translated article is counted in the original category, effectively multiplying the article count for the categories for each translated language.

**Comment by klas** [comment]
Hi,

when you say "article category is not translated", are you referring to the category name that can be optionally displayed in the article parameters?

**Comment by tdzweb** [comment]
Yes. The article category name remains as the original language in various views. It may be related to the issue with the Language Switcher I reported. Something to do with translated aliases perhaps?

To better understand go to the test site I provided above I made an article called "An Article To Be Translated" in the category "Original Category Name", the Norwegian translation is called "The Translated Article" and the translated category name is "Translated Category Name".

You will get 404 errors due to the Language Switcher bug I reported earlier. Just hit "back" and switch language before you drill down.

Notice how the category name in the breadcrumbs is translated correctly.
